backend/models/userModel.js

```diff
@@ -29,7 +29,7 @@ userSchema.methods.matchPassword = function (enteredPassword) {
 // Before we perform the command save, the async function will run
 userSchema.pre("save", async function (next) {
   // This function will automatically run pre save
-
+  console.log("here?");
```

Comment:
It would be good to remove any unnecessary `console.log` statements in your PRs

Reply:
Fixed: Changes are reflected in recent commits